meta/WriteOut.m

```diff
@@ -1256,6 +1263,153 @@
            StringJoin[CreateFormattedSLHABlock /@ sortForBlocks]
           ];
 
+CreateSetDecaysInfoBlockPrototypes[] := "\
```

Comment:
Hmm, actually I'm not so happy about generating such large pieces of C++ source code. Intuitively I'd have put such code into a `.cpp.in` file in `templates/` or so. From my perspective such code snippets would be easier to find there. Are there reasons to generate this code here?

Reply:
The idea probably was that this piece of code might not be needed. If we create model without decays this piece of code will maybe compile but if the complier is smart will not even be emitted.

Reply:
That is of course true. Would you say that it would be more wise to not generate/compile the code at all?